src/main/app/claims/models/claim.ts

```diff
@@ -73,4 +79,12 @@ export default class Claim implements Serializable<Claim> {
 
     return this.remainingDays < 0 && !this.respondedAt
   }
+
+  get isCountyCourtJudgmentSubmitted (): boolean {
+    if (!toBoolean(config.get<boolean>('featureToggles.countyCourtJudgment'))) {
+      return false
```

Comment:
I'm a little worried that checking the toggle here and returning false can give unexpected results, e.g. `else` blocks being executed where we don't want them to.

Reply:
Partly why i suggested moving some of the logic to `isEligibleForCcj`

Reply:
refactored